onyo/cli/tests/test_unset.py

```diff
@@ -173,7 +173,7 @@ def test_unset_key_does_not_exist(repo: OnyoRepo,
 @pytest.mark.parametrize('key', ['num'])
 def test_unset_multiple_assets(repo: OnyoRepo,
                                key: str) -> None:
-    """Test that `onyo unset --keys KEY --asset ASSET [ASSET2 ...]` removes keys from of assets."""
+    r"""Test that `onyo unset --keys KEY --asset ASSET [ASSET2 ...]` removes keys from of assets."""
```

Comment:
A similar complain as above: are the flags up to date, i.e. is it still `--asset`?
Our doc-strings would deserve a bit more love, especially if they would appear in documentation.

Reply:
Thanks for the catch. I have fixed it locally, but will not include it in this PR.